data/rbac-schema/src/main/kotlin/net/corda/rbac/schema/RbacKeys.kt

```diff
@@ -15,4 +15,35 @@ object RbacKeys {
      * Separator for permission string which sits between prefix and the wildcard expression
      */
     const val PREFIX_SEPARATOR = ":"
+
+    /**
+     * Regular expressions to validate common data structures as part of RBAC checks
+     */
+    private const val UUID_CHARS = "[a-fA-F0-9]"
+    const val UUID_REGEX = "$UUID_CHARS{8}-$UUID_CHARS{4}-$UUID_CHARS{4}-$UUID_CHARS{4}-$UUID_CHARS{12}"
```

Comment:
Use "^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[1-5][a-fA-F0-9]{3}-[89aAbB][a-fA-F0-9]{3}-[a-fA-F0-9]{12}$" as per https://github.com/corda/corda-api/blob/release/os/5.0/data/membership-schema/src/main/resources/net/corda/schema/membership/group/policy/1.0/corda.group.policy.json#L22 as UUID actually has more structure than this